src/apps/frontend/types/async-operation.ts

```diff
@@ -0,0 +1,39 @@
+import constant from '../constants';
+
+import { JsonObject } from './common-types';
+
+export class AsyncError {
+  code: string;
+  httpStatusCode: number;
```

Comment:
This is not right. AsyncError cannot have httpStatusCode

Reply:
Removed